.markdownlint.json

```diff
@@ -0,0 +1,7 @@
+{
+  "default": true,
+  "MD002": false,
+  "MD006": false,
+  "MD013": false,
+  "MD033": false
+}
```

Comment:
Why all these exclusions?

Reply:
MD002 - Off by default - MD002 first-heading-h1/first-header-h1 - First heading should be a top level heading
MD006 - Off by default - MD006 ul-start-left - Consider starting bulleted lists at the beginning of the line
MD013 - line-length - Line length
MD033 - no-inline-html - Inline HTML

The first two - I was not sure if the presence of a lint file would default them back to true
The other two - because otherwise there are warnings for things that are not, per se, bad

Reply:
My vscode markdown lint has only MD013 excluded. I'm reasonably happy to exclude MD033 as well. Generally, bare URLs are not too bad, but if you've seen the length of Sharepoint URLs (e.g. `https://centiqltd.sharepoint.com/:w:/s/Centiq/Eb67Av-B3slBtTEBS6pyPrYBPxqwpmC_T18_uaq0Fiu4nQ?e=2AWYr1&isSPOFile=1&sourceId=&params=%7B%22AppName%22%3A%22Teams-Desktop%22%2C%22AppVersion%22%3A%221415%2F21110108720%22%7D`, you'll appreciate the notion of hiding them! The configuration for MD013/033 should be added to the `.vscode/settings.json` file, rather than a separate `.markdownlint.json` file.

```text
"markdownlint.config": {
  "MD013": false,
  "MD033": false
}
```

Reply:
Will do.  I thought it would be clearer for the purposes of a course.